cmake/FindRuby.cmake

```
@@ -13,7 +13,7 @@ if(RUBY_FOUND)
 endif()
 
 find_program(RUBY_EXECUTABLE
-  NAMES ruby2.2 ruby22 ruby2.1 ruby21 ruby2.0 ruby2 ruby1.9.3 ruby193 ruby1.9.2 ruby192 ruby1.9.1 ruby191 ruby1.9 ruby19 ruby1.8 ruby18 ruby
+  NAMES ruby2.5 ruby25 ruby2.4 ruby24 ruby2-3 rubu23 ruby2.2 ruby22 ruby2.1 ruby21 ruby2.0 ruby2 ruby1.9.3 ruby193 ruby1.9.2 ruby192 ruby1.9.1 ruby191 ruby1.9 ruby19 ruby1.8 ruby18 ruby
```

I think there's a typo in here: shouldn't `rubu23` be `ruby23` instead?

```suggestion
  NAMES ruby2.5 ruby25 ruby2.4 ruby24 ruby2-3 ruby23 ruby2.2 ruby22 ruby2.1 ruby21 ruby2.0 ruby2 ruby1.9.3 ruby193 ruby1.9.2 ruby192 ruby1.9.1 ruby191 ruby1.9 ruby19 ruby1.8 ruby18 ruby
```